src/makeInfinityAdder.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let amount = 0;
+
+  return function func(a) {
```

Comment:
perhaps not a good idea for a name of function, maybe better to call it like sum and we can use function expression instead of function declaration  [7.1. Use named function expressions instead of function declarations.](https://github.com/mate-academy/style-guides/blob/master/javascript.md#71-use-named-function-expressions-instead-of-function-declarations)